CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contributing to Puter
+
+Welcome to Puter, the open-source distributed internet operating system. We're excited to have you contribute to our project, whether you're reporting bugs, suggesting new features, or contributing code. This guide will help you get started with contributing to Puter in different ways.
+
+<br>
+
+# Report bugs
+
+Before reporting a bug, please check our [the issues on our GitHub repository](https://github.com/HeyPuter/puter/issues) to see if the bug has already been reported. If it has, you can add a comment to the existing issue with any additional information you have.
+
+If you find a new bug in Puter, please [open an issue on our GitHub repository](https://github.com/HeyPuter/puter/issues). We'll do our best to address the issue as soon as possible. When reporting a bug, please include as much information as possible, including:
+
+- A clear and descriptive title
+- A description of the issue
+- Steps to reproduce the bug
+- Expected behavior
+- Actual behavior
+- Screenshots, if applicable
+- Your operating system and browser
```

Comment:
That's a good idea. 

> Could even include a screenshot.

I need to provide generic information about providing their Puter version since the UI changes quite a bit.